velox/expression/tests/ExpressionFuzzer.h

```diff
@@ -189,6 +189,13 @@ class ExpressionFuzzer {
 
   core::TypedExprPtr getCallExprFromCallable(const CallableSignature& callable);
 
+  /// Return a random signature template mapped to typeName and functionName in
```

Comment:
This comment doesn't seem to match the code. Would you double check?

Reply:
Thanks for catching this. I've updated the comment.